src/main/java/com/patchworkmc/patch/BiomeLayersTransformer.java

```diff
@@ -0,0 +1,48 @@
+package com.patchworkmc.patch;
+
+import org.objectweb.asm.ClassVisitor;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+
+public class BiomeLayersTransformer extends ClassVisitor {
+	private static final String BIOME_LAYERS_NAME = "net/minecraft/class_3645";
+	private static final String PATCHWORK_BIOME_LAYERS_NAME = "net/patchworkmc/api/levelgenerators/PatchworkBiomeLayers";
+
+	public BiomeLayersTransformer(ClassVisitor classVisitor) {
+		super(Opcodes.ASM7, classVisitor);
+	}
+
+	@Override
+	public void visit(int version, int access, String name, String signature, String superName, String[] interfaces) {
+		super.visit(version, access, name, signature, BIOME_LAYERS_NAME.equals(superName) ? PATCHWORK_BIOME_LAYERS_NAME : superName, interfaces);
+	}
+
+	@Override
+	public MethodVisitor visitMethod(int access, String name, String descriptor, String signature, String[] exceptions) {
+		return new MethodTransformer(super.visitMethod(access, name, descriptor, signature, exceptions));
+	}
+
+	private static class MethodTransformer extends MethodVisitor {
+		MethodTransformer(MethodVisitor methodVisitor) {
+			super(Opcodes.ASM7, methodVisitor);
+		}
+
+		@Override
+		public void visitTypeInsn(int opcode, String type) {
```

Comment:
I don't think that anyone will be needing to initialize BiomeLayers? Or at least, it won't need to be redirected.